pyvo/mivot/tests/test_mivot_instance.py

```diff
@@ -39,6 +42,84 @@
             }
 }
 
+test_dict = {
+    "tableref": "Results",
+    "root_object": {
+        "dmid": "_ts_data",
+        "dmrole": "",
+        "dmtype": "cube:NDPoint",
+        "observable": [
+            {
+                "dmtype": "cube:Observable",
+                "dependent": {"value": True},
+                "measure": {
+                    "dmrole": "cube:MeasurementAxis.measure",
+                    "dmtype": "meas:Time",
+                    "coord": {
+                        "dmrole": "meas:Time.coord",
+                        "dmtype": "coords:MJD",
+                        "date": {"value": 1705.9437360200984},
+                    },
+                },
+            },
+            {
+                "dmtype": "cube:Observable",
+                "dependent": {"value": True},
+                "measure": {
+                    "dmrole": "cube:MeasurementAxis.measure",
+                    "dmtype": "meas:GenericMeasure",
+                    "coord": {
+                        "dmrole": "meas:GenericMeasure.coord",
+                        "dmtype": "coords:PhysicalCoordinate",
+                        "cval": {"value": 15.216575},
+                    },
+                },
+            },
+            {
+                "dmtype": "cube:Observable",
+                "dependent": {"value": True},
+                "measure": {
+                    "dmrole": "cube:MeasurementAxis.measure",
+                    "dmtype": "meas:GenericMeasure",
+                    "coord": {
+                        "dmrole": "meas:GenericMeasure.coord",
+                        "dmtype": "coords:PhysicalCoordinate",
+                        "cval": {"value": 15442.456},
+                    },
+                    "error": {
+                        "dmrole": "meas:Measure.error",
+                        "dmtype": "meas:Error",
+                        "statError": {
+                            "dmrole": "meas:Error.statError",
+                            "dmtype": "meas:Symmetrical",
+                            "radius": {"value": 44.15126},
+                        },
+                    },
+                },
+            },
+        ],
+    },
+}
+
+
+@pytest.fixture
+def m_viewer(data_path):
+    return MivotViewer(os.path.join(data_path, "data", "test.mivot_viewer.xml"),
+                       tableref="Results")
+
+
+@pytest.fixture
+def data_path():
+    return os.path.dirname(os.path.realpath(__file__))
```

Comment:
There are some data utilities here: https://docs.astropy.org/en/stable/utils/ref_api.html#file-downloads

Maybe consider using astropy.utils.data.get_pkg_data_filename instead, as we already do that in other modules? Checking it now, it seems that there are a lot of other `os.path.join`s in this module that we didn't spot before, maybe those could be simplified, too (of course, not as part of this PR).

Reply:
Use of `get_pkg_data_filename` in all tests with local data